x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/delete_comment.ts

```diff
@@ -278,6 +286,51 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     });
 
+    describe('attachment stats', () => {
+      it('should set the attachment stats correctly', async () => {
+        const postedCase = await createCase(supertest, postCaseReq);
+
+        await bulkCreateAttachments({
+          supertest,
+          caseId: postedCase.id,
+          params: [postCommentUserReq, postCommentUserReq, postCommentAlertReq],
+          expectedHttpCode: 200,
+        });
+
+        const resolvedCase = await resolveCase({
+          supertest,
+          caseId: postedCase.id,
+        });
+
+        const caseComments = resolvedCase.case.comments!;
+
+        const userComment = caseComments?.find((comment) => comment.type === 'user');
+        const alertComment = caseComments?.find((comment) => comment.type === 'alert');
+
+        await deleteComment({
+          supertest,
+          caseId: postedCase.id,
+          commentId: userComment!.id,
+        });
+
+        await deleteComment({
+          supertest,
+          caseId: postedCase.id,
+          commentId: alertComment!.id,
+        });
+
+        const res = await getCaseSavedObjectsFromES({ es });
+
+        expect(res.body.hits.hits.length).to.eql(1);
+
+        const theCase = res.body.hits.hits[0]._source?.cases!;
+
+        // We added 3 attachments, 2 user and 1 alert and we deleted 1 of each
```

Comment:
nit: Since we call `bulkCreateAttachments`, we could do some checks before deletion and make this clear in the code instead of in a comment.

```
        expect(theCase.total_alerts).to.eql(1);
        expect(theCase.total_comments).to.eql(2);
```

Reply:
This is tested in `x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/bulk_create_attachments.ts`. I thought it was better to focus only on the functionality we are interested in. I added the check.